claims.json

```diff
@@ -2672,8 +2672,9 @@
   "org.apache.kafka *": "apache/kafka",
   "org.apache.kudu kudu-spark2_2.11": "scalacenter/scaladex-void",
   "org.apache.kudu kudu-spark_2.10": "scalacenter/scaladex-void",
-  "org.apache.logging.log4j log4j-api-scala_2.10": "apache/logging-log4j2",
-  "org.apache.logging.log4j log4j-api-scala_2.11": "apache/logging-log4j2",
+  "org.apache.logging.log4j log4j-api-scala_2.10": "apache/logging-log4j-scala",
+  "org.apache.logging.log4j log4j-api-scala_2.11": "apache/logging-log4j-scala",
+  "org.apache.logging.log4j log4j-api-scala_2.12": "apache/logging-log4j-scala",
```

Comment:
I think you are missing api and core: https://github.com/apache/logging-log4j-scala#usage

it would be:

```
"org.apache.logging.log4j log4j-api*": "apache/logging-log4j-scala",
"org.apache.logging.log4j log4j-core*": "apache/logging-log4j-scala",
```